### PR TITLE
Colin Reinhart

### PR DESCRIPTION
**Pull request by ColinReinhart**

### CODE REVIEW

File: final_prep/README.md [hunk 1]
```diff
@@ -52,23 +52,23 @@ When you are finished, add screenshots of your calendar so we can provide feedba
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
-- [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: https://gist.github.com/ColinReinhart/1e8a1b049b01e033b479e3446ef29e29
```
Comment (ameseee) [inline final_prep/README.md:57]: It looks like this isn't finished - no answers starting at the "Special Support Requests" section.

File: final_prep/README.md [hunk 2]
```diff
@@ -23,24 +23,24 @@ Some things to consider as you refactor include...
...
 
-- [ ] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Back End curriculum site](https://backend.turing.edu/)). The detailed classes held on your actual start date may not yet be posted; you can look back at the previous inning to get a feel for what Weeks 1-3 will be like, but know it's not guaranteed to be exactly the same. Also, as you already know, class is held from 9-12 and 1-4MT, Monday-Friday.
-- [ ] Study/Project work time (usually about 25 hours/week outside of class time)
-- [ ] Personal, Health + Wellness
+- [x] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Back End curriculum site](https://backend.turing.edu/)). The detailed classes held on your actual start date may not yet be posted; you can look back at the previous inning to get a feel for what Weeks 1-3 will be like, but know it's not guaranteed to be exactly the same. Also, as you already know, class is held from 9-12 and 1-4MT, Monday-Friday.
+- [x] Study/Project work time (usually about 25 hours/week outside of class time)
+- [x] Personal, Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
```
Comment (ameseee) [inline final_prep/README.md:46]: I don't see calendar screen shots below - are there somewhere else within this repository?

File: final_prep/README.md [hunk 3]
```diff
@@ -52,23 +52,23 @@ When you are finished, add screenshots of your calendar so we can provide feedba
...
+  - [x] Add link to your gist here: https://gist.github.com/ColinReinhart/1e8a1b049b01e033b479e3446ef29e29
 
 ## Lesson Prep
 
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [x] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [x] Add a link to your gist here: https://gist.github.com/ColinReinhart/92690fb65b8b21ef480db46f0510769e
```
Comment (ameseee) [inline final_prep/README.md:64]: This is awesome, the color coding is a great strategy!

File: section2/exercises/if_statements.rb [hunk 4]
```diff
@@ -63,3 +64,32 @@
...
+has_sauce = true
+if cups_of_flour >= 2 == has_sauce
+  puts "I can make a pizza"
+else
+  puts "I cannot make pizza"
+end
+
+cups_of_flour = 3
+has_sauce = true
+if cups_of_flour >= 2 == has_sauce
```
Comment (ameseee) [inline section2/exercises/if_statements.rb:91]: Nice solution, it's good to see you tested it with various values in each variable.

**Review by ameseee — COMMENTED** [review]
@ColinReinhart your work with Ruby is really strong! The bigger goal of Mod 0 is the professional habits and mindset. Please check the first comments RE calendar and mentor gist and get those in, and keep in mind your timeliness/taking responsibility when not in future assignments.

File: final_prep/README.md (hunk 2 above)
Comment (ColinReinhart) [reply final_prep/README.md:46]: I have added the screenshots.

File: final_prep/README.md (hunk 1 above)
Comment (ColinReinhart) [reply final_prep/README.md:57]: I have added these answers.

**Pull request closed without merging**